exec/cnex/nstring.c

```diff
@@ -157,6 +202,17 @@ TString *string_appendString(TString *s, TString *ns)
     return s;
 }
 
+TString *string_fillString(TString *s, char c, size_t count)
+{
+    s->data = realloc(s->data, s->length + count);
```

Comment:
So this `realloc()` seems to indicate that this function is going to *append* the given number of characters, right? That's why you are computing `s->length + count`? But then the implementation just smacks the filled characters over top of the *beginning* of the given string, leaving some of the newly allocated space at the end of the buffer uninitialized. That doesn't seem right.

Reply:
Well, what do you *need* it to do? Don't do anything more than what you need at this point.